pallets/pallet-bonded-coins/src/curves/lmsr.rs

```diff
@@ -1,3 +1,22 @@
+/// Implementation of the [Logarithmic Market Scoring Rule](https://mason.gmu.edu/~rhanson/mktscore.pdf) (LMSR) bonding curve.
+/// The cost of purchasing an amount of assets from the market is determined using the liquidity parameter of the
+/// LMSR model and the current supply of assets in the market.
+/// The LMSR bonding curve is defined by the following equation:
+///
+/// C(s) = m * ln(Σ(e^(s_i/m))),
+///
+/// where:
+/// - `s` is the supply of assets,
+/// - `m` is the liquidity parameter of the LMSR model,
+/// - `s_i` is the current supply of the assets.
```

Comment:
can you make the difference between s and s_i clearer? Is s the cumulated total supply of all assets, and s_i the supply of one asset?

Reply:
There was a little mistake. :grimacing:  Fixed in: [8f8664a](https://github.com/KILTprotocol/kilt-node/pull/801/commits/8f8664a534ff38f5e7f491145f18f69f0eb1019f)